### ISSUE TITLE
[Popup] potential for focusgroup alignment?

### ISSUE THREAD

**Issue by travisleithead** [openui/open-ui#401]
Hi folks! 

A new [`focusgroup` attribute](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/Focusgroup/explainer.md) proposal was released recently, and while it and the Popup explainer evolved independently, I wonder if there is an opportunity for one to leverage another? (Confession: I've not followed the Popup work very closely but happy to get reactions, comments, concerns, etc. on whether `focusgroup` works for the Popup use case or not.

Much thanks!

**Comment by bkardell** [comment]
Yeah, as I suggested on twitter I think this is even bigger than just popup.  Our spicy-sections have this in tabview - and the underlying magic we have been holding a bit on as @tabatkins and @mirisuzanne had a proposal they were working on in CSS for toggle which also overlaps... I definitely think we should align.

**Comment by travisleithead** [comment]
I read through [Toggle](https://tabatkins.github.io/css-toggle/) (assume that's the right URL). This idea has been brewing in Tab's head for a [very long time](https://www.xanthir.com/b4Kn0)!

Focusgroup deals first with focus (and omits toggling logic as a non-goal), and toggle handles the "selection" or "picking" (then begins to extend into focus). I think the two could work well together. However, there are some places where they intersect--around creating/linking groups--where the specs' opinions differ. I also would like to see the platform not have two ways of creating groups/scopes--so reconciling that aspect seems important. Finally, I note that "multi-select" (as far as use cases go for things like multi-select combo/list-boxes) is not supported in Toggle.

**Comment by mfreed7** [comment]
So I think this issue has really evolved to be a question about the intersection between Toggle and FocusGroup, is that right? As such, I've renamed it just a bit.

**Comment by mirisuzanne** [comment]
That seems accurate. I think at this point the two proposals are entirely non-overlapping: Toggle 'groups' are entirely distinct from focus 'groups' in terms of their current behavior. However, the use-cases have a lot of overlap. When using toggle groups, you likely _want_ focus groups as well. So I think the main issue here is something like:

- Should the toggle feature allow an author to group focus in addition to grouping state?
- If so, that would mean focus-groups need to be controlled by CSS (at least implicitly created by toggle groups)
- Which raises the question if they can (or should) be __explicitly_/_exclusively_ controlled in CSS?

**Comment by travisleithead** [comment]
With CSS Toggles [no longer being actively pursued](https://docs.google.com/document/d/10JxXgeDAJ5zerOdxoWM9ZU9d5iMOdzGJt0EhlhkOkxM/edit) I'm going to close this. There are certainly opportunities for alignment with other proposals, and we can track those separately. Re-open if folks disagree.
